telemetry/service.go

```diff
@@ -0,0 +1,84 @@
+package telemetry
+
+import (
+	"context"
+	"encoding/json"
+	"github.com/NethermindEth/juno/utils"
+	"github.com/libp2p/go-libp2p/core/peer"
+	"nhooyr.io/websocket"
+	"runtime"
+	"time"
+)
+
+var startTime = time.Now()
+
+type service struct {
+	peerID       peer.ID
+	version      string
+	websocketURL string
+
+	log  utils.SimpleLogger
+	conn *websocket.Conn
+}
+
+func NewService(peerID peer.ID, websocketURL, version string) *service {
+	return &service{
+		peerID:       peerID,
+		version:      version,
+		websocketURL: websocketURL,
+	}
+}
+
+func (s *service) Run(ctx context.Context) error {
+	ticker := time.NewTicker(5 * time.Second)
+	defer ticker.Stop()
+
+	for {
+		select {
+		case <-ctx.Done():
+			return nil
+		case <-ticker.C:
+			err := s.sendTelemetry(ctx)
+			if err != nil {
+				s.log.Errorw("Failed to send telemetry", "err", err)
+			}
```

Comment:
Do you think we should sleep for a bit before the next iteration of the loop?

Reply:
I thought that ticker should do that, though if sending takes 5 sec next request will start immediately. We can update ticker once request is complete, wdyt?